.vscode/settings.json

```diff
@@ -1,3 +1,6 @@
 {
-  "git.ignoreLimitWarning": true
+  "git.ignoreLimitWarning": true,
+  "yaml.schemas": {
+    " https://raw.githubusercontent.com/garethr/openshift-json-schema/master/v4.1.0/_definitions.json": ["/*"],
```

Comment:
This fixed the YAML plugin if you have it installed so that it doesn't display errors for yaml files.  Still don't think the plugin works well for intelli-sense however.